src/mlpack/tests/feedforward_network_2_test.cpp

```diff
@@ -77,7 +78,8 @@ TEST_CASE("RBFNetworkTest", "[FeedForwardNetworkTest]")
   }
 
   arma::mat testData;
-  data::Load("thyroid_test.csv", testData, true);
+  if (!data::Load("thyroid_test.csv", testData, true)) 
```

Comment:
```suggestion
  if (!data::Load("thyroid_test.csv", testData)) 
```
Third argument not required.